antidox/perspective.py

```diff
@@ -170,71 +158,101 @@ def main(argv):
   parser.add_argument('--csv_file', help='choose CSV file to process')
   parser.add_argument('--wiki_pagename', help='insert the talk page name')
   parser.add_argument('--content', help='specify a column in dataset to retreive data from')
-  parser.add_argument('--output', help='path for output file')
-  parser.add_argument('--suffix', help='output file suffix')
-  parser.add_argument('--project', help='project id for bigquery table')
-  args = parser.parse_args(argv)
+  parser.add_argument('--output', help='path for output file in cloud bucket')
+  parser.add_argument('--nd_output', help='gcs path to store ndjson results')
+  parser.add_argument('--project', help='project id for bigquery table', \
+                                   default='wikidetox-viz')
+  parser.add_argument('--gproject', help='gcp project id')
+  parser.add_argument('--temp_location', help='cloud storage path for temp files \
+                                              must begin with gs://')
+  args, pipe_args = parser.parse_known_args(argv)
   apikey_data, perspective, dlp = get_client()
-
-
-if __name__ == '__main__':
-  main(sys.argv[1:])
-  with beam.Pipeline(options=PipelineOptions()) as pipeline:
+  options = PipelineOptions(pipe_args)
```

Comment:
NIT: there are about 9 lines here for setting the options, you might consider moving this to a new function like `def get_pipeline_options()` that returns this object